install/kubernetes/istio-auth.yaml

```diff
@@ -817,11 +813,11 @@ apiVersion: "config.istio.io/v1alpha2"
 kind: prometheus
 metadata:
   name: handler
-  namespace: istio-config-default
+  namespace: istio-system
 spec:
   metrics:
   - name: request_count
-    instance_name: requestcount.metric.istio-config-default
+    instance_name: requestcount.metric.istio-system
```

Comment:
@mandarjog can the namespace domain be inferred here?

Reply:
it cannot at this point. this is a code change to the prometheus adapter (or elsewhere in Mixer).

Reply:
https://github.com/istio/mixer/issues/1323